test/shared/src/main/scala/zio/test/mock/TestConsole.scala

```diff
@@ -46,9 +47,19 @@ case class TestConsole(ref: Ref[TestConsole.Data]) extends Console.Service[Any]
           }
     } yield input
   }
+
+  def feedLine(line: String): UIO[Unit] =
+    ref.update(data => data.copy(input = line :: data.input)).unit
+
+  val output: UIO[Vector[String]] =
```

Comment:
This is good. Probably good to add a `clearOutput`, too, and maybe `clearInput`.